src/PFire.Core/PFireServer.cs

```diff
@@ -84,6 +93,7 @@ public XFireClient GetSession(User user)
 
         private void AddSession(XFireClient session)
         {
+            session.Server = this;
```

Comment:
Is there a way we can do this when `XFireClient` is initially created instead of here?